SDK forward and backward compatibility (#4036)
dapr/docs: daprdocs/content/en/operations/support/support-release-policy.md

| Mar 25th 2022 | 1.6.1</br>   | 1.6.0 | Java 1.4.0 </br>Go 1.3.1 </br>PHP 1.1.0 </br>Python 1.5.0 </br>.NET 1.6.0 </br>JS 2.0.0 | 0.9.0 | Unsupported |  |
| Jan 25th 2022 | 1.6.0</br>   | 1.6.0 | Java 1.4.0 </br>Go 1.3.1 </br>PHP 1.1.0 </br>Python 1.5.0 </br>.NET 1.6.0 </br>JS 2.0.0 | 0.9.0 | Unsupported |  |

> ## SDK forward and backward compatibility

Reviewer: ```suggestion
## SDK compatibility
The SDKs and runtime are committed to non-breaking changes other than those required for security issues.  All breaking changes are announced if required in the release notes. 
```

> **Forward compatibility**  

Reviewer: ```suggestion
**SDK and runtime forward compatibility**  
```

Newer Dapr SDKs support the latest version of Dapr runtime and two previous versions (N-2). 

> **Backward compatibility**  

Reviewer: ```suggestion
**SDK and runtime backward compatibility**  
```

> With non-breaking changes, the current SDK version and three previous three versions (N-3)are supported for a newer Dapr runtime version.

Reviewer: ```suggestion
For a new Dapr runtime, the current SDK version and two previous versions (N-2) are supported. 
```
